manifests/install.pp

```diff
@@ -13,8 +13,8 @@
       }
       Anchor['filebeat::install::begin'] -> Class['filebeat::install::linux'] -> Anchor['filebeat::install::end']
       if $::filebeat::manage_repo {
-        class { '::filebeat::repo': }
-        Class['filebeat::repo'] -> Class['filebeat::install::linux']
+        include elastic_stack::repo
+        Class['elastic_stack::repo'] -> Class['filebeat::install::linux']
```

Comment:
I love the idea of using the upstream elastic repo as suggested in #189, but we customize a few things in the setup. I think it's reasonable to ask users that need those customizations to set `manage_repo` to false and manage it themselves, but we should probably remove references to the parameters too (README, params.pp, init.pp).